<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionarMaterial</class>
 <widget class="QWidget" name="GestionarMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Material</string>
  </property>
  <widget class="QLineEdit" name="filterLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Filtrar material...</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>660</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnadir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Añadir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModificar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>